helm/secdash/dags/mark_provisioned.py

```diff
@@ -40,7 +39,12 @@ def fetch_products_mark_completed(provisioner_api_url, mark_provisioned_url, mon
             workflow_name = f"{cluster}-{licence_plate}-{request_id}"
             provisioner_workflow_url = f"{provisioner_api_url}/{workflow_name}"
             request_status_in_provisioner = requests.get(provisioner_workflow_url).json()
-            request_phase = request_status_in_provisioner['metadata']['labels']['workflows.argoproj.io/phase']
+            if 'metadata' not in request_status_in_provisioner:
```

Comment:
if `metadata` does not exists, let's just exist the function.

Reply:
When I was testing, it printed an error message:
```
{
    "code": 5,
    "message": "workflows.argoproj.io \"klab-dc5249-666b6bbd9d3e7453bbad1e66\" not found"
}
```
 so I thought it would be useful for understanding what went wrong if metadata is not available.